src/repart/repart.c

```diff
@@ -7305,6 +7315,10 @@ static int resolve_copy_blocks_auto(
                 try1 = "/";
         else if (type.designator == PARTITION_USR_VERITY)
                 try1 = "/usr/";
+        else if (type.designator == PARTITION_ROOT_VERITY_SIG)
+                try1 = "/dev/disk/by-designator/root-verity-sig";
+        else if (type.designator == PARTITION_USR_VERITY_SIG)
+                try1 = "/dev/disk/by-designator/usr-verity-sig";
```

Comment:
oh, you are thinking the auto discovery was smarter than it actually is...

this is much dumber, you can just go by / and /usr/ here too, we do not actually check at all what's used: the algorithm just takes the first matching partition off the disk backing those dirs.

or in other words, if you look below, we issue find_backing_devno() to find the backing device node of these dirs here, but once we found it, we just trace it to there originating whole device, then forget all context and simply look for the first matching partition by its type uuid on it.

We should probably make this smarter (as in an A/B scenario this will always pick A), but that's a different story from just making verity sig work.